### PR TITLE
FEM lab: Add separate frames layout options

### PR DESCRIPTION
**Pull request by goplayoutside3**
Staging branch URL: https://pr-6536.pfe-preview.zooniverse.org

I added four options to the FEM lab's multi-image subject options section: "row", "grid2", and "grid3" are already established as layout options in PFE. In this PR I also added the option of "col". The layout options only appear when a workflow has "allow volunteers to switch between flipbook and separate frames" enabled:

Screenshot 2023-04-17 at 5 02 45 PM

Below are the four layouts rendered in a workflow with a subject that has five frames. The subject image becomes tiny in a "row" or "3-col grid", but we can trust project teams will choose the layout that best suits their subject type. This is just an example.

Column
col

Row
row

Grid With 2 Columns
grid2

Grid With 3 Columns
grid3

@seanmiller26 for your review in addition to the screenshots above:
    - Should I add text to the lab section explaining that the layout will always be one column when volunteers use a small screen such as a mobile device?

# Review
To review you can visit my flipbook test project's workflow in the FEM lab: https://local.zooniverse.org:3735/lab/19645/workflows/22532?env=production&femLab=true

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `npm ci` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on GitHub Actions?

### CODE REVIEW

**Comment by seanmiller26** [comment]
I know we are leaving the final decision to the teams but I'd like to still put recommendations next to these options. So the final copy would be: 
_Large font_ Single column _small font_ - all frames stacked vertically (recommended only for portrait subjects) 
Single row - all frames side by side horizontally (recommended only for landscape subjects)

> The subject image becomes tiny in a "row" or "3-col grid", but we can trust project teams will choose the layout that best suits their subject type.

I am really curious what happens in responsive situations, especially the row option. How small can subjects get and do we want to impose a restriction at that point? Just something to think about. Below we mention it defaults to a single row. Would that be at the 768px breakpoint then?

> Should I add text to the lab section explaining that the layout will always be one column when volunteers use a small screen such as a mobile device?

Yes - 'The layout will default to a single column when volunteers use a small screen such as a mobile device'

**Comment by goplayoutside3** [comment]
@seanmiller26 

Here's a screenshot of the updated text:
Screenshot 2023-04-19 at 1 57 52 PM

Responsiveness will be handled in the FEM codebase. Once this PR for the lab options is merged, I'll tag you for review in the FEM PR ([#4481](https://github.com/zooniverse/front-end-monorepo/pull/4481)). For instance, we'll need to decide what to use as a breakpoint, whether it's the subject image width, classifier layout orientation, or volunteer's device width.

**Comment by seanmiller26** [comment]
Final copy: 

- Small change to parenthesis
Single column - (recommended for landscape subjects; default for mobile devices)

- Remove parenthesis and match small font
Grid - frames distributed evenly over x columns

Otherwise looks good!

File: app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx
Comment (eatyourgreens) [inline app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx:66]: A small thing but labels should be short. I'd put the supplementary text outside the label, to make the label more accessible.

File: app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx
Comment (eatyourgreens) [inline app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx:82]: Same comment here about keeping labels short and accessible.

File: app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx
Comment (eatyourgreens) [inline app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx:93]: A shorter label would be better here too.

File: app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx
Comment (eatyourgreens) [inline app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx:104]: Move supplementary text out of the label here.

File: app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx
Comment (eatyourgreens) [inline app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx:115]: Move supplementary text out of the label here.

**Review by eatyourgreens — COMMENTED** [review]
If it isn't too much of a problem, I'd use short, accessible labels for the inputs.

**Review by seanmiller26 — APPROVED** [review]

**Pull request merged**
